### PR TITLE
Add default icon for aggregated view and UI Enhancement

### PR DESCRIPTION
**Pull request by zhyuanqi**
### Description

Add default icon for aggregated view and add default icon

## Screenshot

Screenshot 2024-04-09 at 8 57 53 PM

### Check List

- [ ] All tests pass
  - [ ] `yarn test:jest`
  - [ ] `yarn test:jest_integration`
- [ ] New functionality includes testing.
- [ ] New functionality has been documented.
- [ ] Update [CHANGELOG.md](./../CHANGELOG.md)
- [ ] Commits are signed per the DCO using --signoff

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -83,6 +83,7 @@ Inspired from [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 - [Multiple Datasource] Pass selected data sources to plugin consumers when the multi-select component initially loads ([#6333](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/6333))
 - [Multiple Datasource] Add installedPlugins list to data source saved object ([#6348](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/6348))
 - [Multiple Datasource] Add default icon in multi-selectable picker ([#6357](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/6357))
+- [Multiple Datasource] Add default icon for aggregated view and add default icon ([#6386](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/6386))
```
Comment (BionIT) [inline CHANGELOG.md:86]:
```suggestion
- [Multiple Datasource] Add default icon for aggregated view ([#6386](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/6386))
```

File: src/plugins/data_source_management/public/components/data_source_aggregated_view/data_source_aggregated_view.tsx [hunk 2]
```diff
@@ -166,7 +162,27 @@ export class DataSourceAggregatedView extends React.Component<
           panelPaddingSize="none"
           anchorPosition="downLeft"
         >
-          <EuiContextMenu initialPanelId={0} panels={panels} />
+          <EuiContextMenuPanel title="Data sources">
+            <EuiPanel color="transparent" paddingSize="s" style={{ width: '300px' }}>
```
Comment (BionIT) [inline src/plugins/data_source_management/public/components/data_source_aggregated_view/data_source_aggregated_view.tsx:166]: let's extract style `width` to scss file

File: src/plugins/data_source_management/public/components/data_source_aggregated_view/data_source_aggregated_view.tsx [hunk 3]
```diff
@@ -166,7 +162,27 @@ export class DataSourceAggregatedView extends React.Component<
...
+              <EuiSpacer size="s" />
+              <EuiSelectable
+                options={items}
+                data-test-subj={'dataSourceSelectable'}
```
Comment (BionIT) [inline src/plugins/data_source_management/public/components/data_source_aggregated_view/data_source_aggregated_view.tsx:170]: we should be more specific of the name

File: CHANGELOG.md (hunk 1 above)
Comment (zhyuanqi) [reply CHANGELOG.md:86]: WIll update this.

File: src/plugins/data_source_management/public/components/data_source_aggregated_view/data_source_aggregated_view.tsx [hunk 4]
```diff
@@ -166,7 +162,27 @@ export class DataSourceAggregatedView extends React.Component<
...
+                renderOption={(option) => (
+                  <EuiFlexGroup alignItems="center">
```
Comment (BionIT) [inline src/plugins/data_source_management/public/components/data_source_aggregated_view/data_source_aggregated_view.tsx:172]: looks like we are duplicating ourselves now, found https://github.com/opensearch-project/OpenSearch-Dashboards/blob/b4130aa2c1c80033d1bec4d18d44e20612ac0784/src/plugins/data_source_management/public/components/data_source_selector/data_source_selector.tsx#L176

**Pull request closed without merging**